packages/office-ui-fabric-react/src/components/Theme/ThemePage.tsx

```diff
@@ -115,7 +115,7 @@ export class ThemePage extends React.Component<any, any> {
   private _onColorChanged(index: number, newColor: string) {
     let { colors } = this.state;
     let color = colors[index];
-    let theme: ITheme = {};
+    let theme: Partial<IPalette> = {};
```

Comment:
Huh, you're right. Can you rename this variable to `pallete` or something? Then you can just `loadTheme({ pallete })

Also you're missing a change request for the styling package, so it wouldn't get published. maybe we need to add the check step.